src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -14,7 +14,7 @@ public static partial class Http
 
             public static string SecureHost => GetValue("COREFX_SECUREHTTPHOST", DefaultAzureServer);
 
-            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "http2.akamai.com");
+            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "www.microsoft.com");
```

Comment:
Since there are differences between how these implement HTTP/2, seems like we should have tests that use both rather than just replacing it.